AIDriver.lua

```diff
@@ -1193,9 +1193,11 @@ function AIDriver:dischargeAtUnloadPoint(dt,unloadPointIx)
 				z = courseplay:isNodeTurnedWrongWay(vehicle,tipRefpoint)and -z or z
 
 				local foundHeap = self:checkForHeapBehindMe(tipper)
```

Comment:
This one would be: `local foundHeap,fillLevel = self:checkForHeapBehindMe(tipper)`

Reply:
I've made those changes in latest commit

Reply:
@MddMBorg Nice work. Thank you :)